src/main/java/net/minecraftforge/common/extensions/IForgeDispensibleContainerItem.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.common.extensions;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.DispensibleContainerItem;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.phys.BlockHitResult;
+import org.jetbrains.annotations.Nullable;
+
+public interface IForgeDispensibleContainerItem
+{
+    private DispensibleContainerItem self()
+    {
+        return (DispensibleContainerItem)this;
+    }
+
+    /**
+     * ItemStack sensitive version of {@link DispensibleContainerItem#emptyContents(Player, Level, BlockPos, BlockHitResult)}
+     * Tries to place the content of the container into the level as a block.
+     *
+     * @param player    Player who places the fluid. May be null for blocks like dispensers.
+     * @param level     Level to place the content in
+     * @param pos       The position in the level to place the content
+     * @param hitResult Hit result of the interaction. May be null for blocks like dispensers.
+     * @param container ItemStack of the container. May be null for backwards compatibility.
+     * @return true if the placement was successful, false otherwise
```

Comment:
Same thing here, frame it as emptying the contents of the container and not placing the contents in the level.

Reply:
done